src/ui/settings/components/SettingsTooltip.tsx

```diff
@@ -19,9 +19,11 @@ interface OwnProps {
 }
 
 const SettingsTooltip: React.FunctionComponent<OwnProps> = ({ hrefURL }) => {
-  const link = (hrefURL.startsWith('http') ? hrefURL : `https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/wiki/Documentation${hrefURL}`);
+  const link = hrefURL.startsWith('http')
+    ? hrefURL
+    : `https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/wiki/Documentation${hrefURL}`;
   return (
-    <a href={link} target={'_blank'} className="tooltipCustom">
+    <a href={link} target="_blank'" rel="noreferrer" className="tooltipCustom">
```

Comment:
```suggestion
    <a href={link} target="_blank" rel="noreferrer" className="tooltipCustom">
```
Removing the lone single quote.  should be `_blank` for target I believe